app/src/main/java/org/mozilla/focus/home/HomeScreenBackground.java

```diff
@@ -64,12 +61,29 @@ void init() {
     @Override
     protected void onDraw(Canvas canvas) {
         super.onDraw(canvas);
-        canvas.drawRect(0, 0, getWidth(), getHeight(), paint);
+        if (!isNight) {
+            canvas.drawRect(0, 0, getWidth(), getHeight(), paint);
+        } else {
+            // Add status bar's height as a padding on top to let HomeFragment star background align with TabTrayFragment
+            setPadding(0, ViewUtils.getStatusBarHeight(((Activity) getContext())), 0, 0);
```

Comment:
Same reason why we don't have to reset this as previous review. 
I think it's more appropriate to set the padding here since its size depends on status bar's height which may slightly change over different SDK version. At least, status bar's height is 25 dp before Android M and is 24 dp after Android M.

Reply:
Nice point!